pkg/queue/concurrency_state.go

```diff
@@ -22,17 +22,22 @@ import (
 	"net/http"
 	"os"
 	"sync"
+	"time"
 
 	"go.uber.org/atomic"
 	"go.uber.org/zap"
 )
 
+//nolint:gosec // Filepath, not hardcoded credentials
+const ConcurrencyStateTokenVolumeMountPath = "/var/run/secrets/tokens"
```

Comment:
`ConcurrencyStateTokenVolumeMountPath` was moved to [`pkg/reconciler/revision/resources/deploy.go`](https://github.com/knative/serving/blob/main/pkg/reconciler/revision/resources/deploy.go#L40) in a previous commit, so this line can be removed.

Reply:
changed